test/tools/ossfuzz/protomutators/YulProtoMutator.cpp

```diff
@@ -0,0 +1,2081 @@
+					auto ifCond = new Expression();
+					ifCond->set_allocated_unop(notOp);
+					_message->set_allocated_cond(ifCond);
+				}
+			},
+			_message,
+			_seed,
+			YPM::s_mediumIP,
+			"If condition inverted"
+		);
+	}
+);
+
+// Remove inverted condition in if statement
+static YPR<IfStmt> revertIfCondition(
+	[](IfStmt* _message, unsigned int _seed)
+	{
+		YPM::functionWrapper<IfStmt>(
+			[](IfStmt* _message, YulRandomNumGenerator&)
+			{
+				if (_message->has_cond() && _message->cond().has_unop() &&
+						_message->cond().unop().has_op() && _message->cond().unop().op() == UnaryOp::NOT)
+				{
+					auto oldCondition = _message->release_cond();
+					auto unop = oldCondition->release_unop();
+					auto conditionWithoutNot = unop->release_operand();
+					_message->set_allocated_cond(conditionWithoutNot);
+					delete (oldCondition);
+					delete (unop);
+				}
+			},
+			_message,
+			_seed,
+			YPM::s_mediumIP,
+			"If condition reverted"
+		);
+	}
+);
+
+// Append break statement to a statement block
+static YPR<Block> addBreakStmt(
+	[](Block* _message, unsigned int _seed)
+	{
+		YPM::functionWrapper<Block>(
+			[](Block* _message, YulRandomNumGenerator&)
+			{
+				_message->add_statements()->set_allocated_breakstmt(new BreakStmt());
+			},
+			_message,
+			_seed,
+			YPM::s_highIP,
+			"Break statement added"
+		);
+	}
+);
+
+// Remove break statement in body of a for-loop statement
+static YPR<ForStmt> removeBreakStmt(
+	[](ForStmt* _message, unsigned int _seed)
+	{
+		YPM::functionWrapper<ForStmt>(
+			[](ForStmt* _message, YulRandomNumGenerator&)
+			{
+				if (_message->has_for_body())
+					for (auto& stmt: *_message->mutable_for_body()->mutable_statements())
+						if (stmt.has_breakstmt())
+						{
+							delete stmt.release_breakstmt();
+							stmt.clear_breakstmt();
+							break;
+						}
+			},
+			_message,
+			_seed,
+			YPM::s_mediumIP,
+			"Break statement removed"
+		);
+	}
+);
+
+// Add continue statement to statement block.
+static YPR<Block> addContStmt(
+	[](Block* _message, unsigned int _seed)
+	{
+		YPM::functionWrapper<Block>(
+			[](Block* _message, YulRandomNumGenerator&)
+			{
+				_message->add_statements()->set_allocated_contstmt(new ContinueStmt());
+			},
+			_message,
+			_seed,
+			YPM::s_highIP,
+			"Continue statement added"
+		);
+	}
+);
+
+/// Remove continue statement from for-loop body
+static YPR<ForStmt> removeContinueStmt(
+	[](ForStmt* _message, unsigned int _seed)
+	{
+		YPM::functionWrapper<ForStmt>(
+			[](ForStmt* _message, YulRandomNumGenerator&)
+			{
+				if (_message->has_for_body())
+					for (auto& stmt: *_message->mutable_for_body()->mutable_statements())
+						if (stmt.has_contstmt())
+						{
+							delete stmt.release_contstmt();
+							stmt.clear_contstmt();
+							break;
+						}
+			},
+			_message,
+			_seed,
+			YPM::s_mediumIP,
+			"Continue statement removed"
+		);
+	}
+);
+
+/// Mutate expression into an s/m/calldataload
+static YPR<Expression> addLoadZero(
```

Comment:
`add` sounds like it's appending a new expression. Maybe `setToLoadZero` or similar?